eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/RequestResponseChannel.java

```diff
@@ -149,11 +149,13 @@ private void onLinkCloseComplete(final Exception exception) {
 
         if (closeRefCount.decrementAndGet() <= 0)
             if (exception == null) {
-                onClose.onComplete(null);
+                if (onClose != null)
+                    onClose.onComplete(null);
                 if (onGraceFullClose != null)
                     onGraceFullClose.onComplete(null);
             } else {
-                onClose.onError(exception);
+                if (onClose != null)
+                    onClose.onError(exception);
                 if (onGraceFullClose != null)
                     onGraceFullClose.onError(exception);
```

Comment:
Doesn't checkstyle complain about no braces here? If not, why not?

Reply:
No checkstyle complaint about no braces here. I am not sure about if the CheckStyle rule has set up correctly or not. Will check why it happens and will add braces around the if statement for sure.